src/transform.ts

```diff
@@ -98,7 +129,7 @@ function transformJSONSchemaToFakerCode(jsonSchema?: OpenAPIV3.SchemaObject, key
       return transformStringBasedOnFormat(jsonSchema.format, key);
     case 'number':
     case 'integer':
-      return `faker.datatype.number({ min: ${jsonSchema.minimum}, max: ${jsonSchema.maximum} })`;
+      return transformNumberBasedOnSchema(jsonSchema);
```

Comment:
`transformNumberBasedOnSchema` creates an object for the min/max values and runs it through `JSON.stringify`. This strips out undefined values, keeping the mock cleaner.